src/Features/CSharp/Portable/Completion/CompletionProviders/TupleNameCompletionProvider.cs

```diff
@@ -75,10 +79,16 @@ private void AddItems(ImmutableArray<INamedTypeSymbol> inferredTypes, int index,
                 }
 
                 var field = type.TupleElements[index];
-                var item = CommonCompletionItem.Create(
-                    field.Name + ":", _cachedRules, Glyph.FieldPublic);
+                var item = CommonCompletionItem.Create(field.Name + ColonString, _cachedRules, Glyph.FieldPublic);
```

Comment:
Thanks for the clarification @CyrusNajmabadi.
I noticed that completion of named parameters does something special, using the name (without colon) as "filterText". Should the completion for tuple names do the same?